apps/admin_panel/assets/src/app/bootup/action.js

```diff
@@ -0,0 +1,7 @@
+import createHeaders from '../../utils/headerGenerator'
```

Comment:
I think websockets should connect only after login, not on app initialization.

Reply:
it does connect only on login, this action called in `authenticationRoute`